### PR TITLE
Add virtual machines to style guide.

### PR DESCRIPTION
**Pull request by cgcardona**

### CODE REVIEW

**Review by yulin-dong — DISMISSED** [review]

**Review by yulin-dong — CHANGES_REQUESTED** [review]
Subnet-EVM should be removed from the paragraph above.

**Review by yulin-dong — APPROVED** [review]

File: style-guide.md [hunk 1]
```diff
@@ -13,11 +13,20 @@ or logs.
...
-- Coreth for <https://github.com/ava-labs/coreth>
-- Subnet-EVM for <https://github.com/ava-labs/subnet-evm>
 - Avalanche-CLI for <https://github.com/ava-labs/avalanche-cli>
 - Avalanche Network Runner for <https://github.com/ava-labs/avalanche-network-runner>
 
+### Virtual Machines
+
+- Coreth for <https://github.com/ava-labs/coreth>
+- Subnet-EVM for <https://github.com/ava-labs/subnet-evm>
+- SpacesVM <https://github.com/ava-labs/spacesvm>
```
Comment (yulin-dong) [inline style-guide.md:23]:
```suggestion
- SpacesVM for <https://github.com/ava-labs/spacesvm>
```

File: style-guide.md [hunk 2]
```diff
@@ -13,11 +13,20 @@ or logs.
...
+- BlobVM <https://github.com/ava-labs/blobvm>
```
Comment (yulin-dong) [inline style-guide.md:24]:
```suggestion
- BlobVM for <https://github.com/ava-labs/blobvm>
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: style-guide.md (hunk 1 above)
Comment (cgcardona) [reply style-guide.md]: Resolved

**Review by yulin-dong — APPROVED** [review]

File: style-guide.md (hunk 2 above)
Comment (cgcardona) [reply style-guide.md]: Resolved

**Pull request merged**